### PR TITLE
Retrying new files for Philippines

### PR DESCRIPTION
**Pull request by jakeekaj**
@casey-collab Please check

### CODE REVIEW

File: Philippines/bago/bagongpanimula.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Dito nag-uumpisa ang iyong daan tungo sa pagbabago.
+
+Gusto mo bang [mag-aral](Philippines/aral.md) pa?
```
Comment (casey-collab) [inline Philippines/bago/bagongpanimula.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: Philippines/salamat.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 Maligayang pasko at manigong bagong taon sa inyo! Salamat sa Udacity para sa
 
-online class na ito :D
\ No newline at end of file
+online class na ito :D
+
+Pumunta [dito](Philippines/bago/bagongpanimula.md) para sa bagong istorya.
```
Comment (casey-collab) [inline Philippines/salamat.md]: This link is also invalid.

File: Philippines/bago/aral/aral.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Kung gusto mo pa mag-aral. eh di bumalik ka sa Udacity at kunin lahat ng kursong gusto mo. Good luck sa iyo!
```
Comment (casey-collab) [inline Philippines/bago/aral/aral.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: Philippines/bago/bagongpanimula.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+Dito nag-uumpisa ang iyong daan tungo sa pagbabago.
```
Comment (casey-collab) [inline Philippines/bago/bagongpanimula.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by jakeekaj** [comment]
@casey-collab

File: Philippines/bago/bagongpanimula.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
...
+
+Gusto mo bang [mag-aral](Philippines/bago/aral/aral.md) pa?
```
Comment (casey-collab) [inline Philippines/bago/bagongpanimula.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: Philippines/salamat.md (hunk 2 above)
Comment (casey-collab) [inline Philippines/salamat.md]: This link is also invalid.

File: Philippines/bago/aral/aral.md (hunk 3 above)
Comment (casey-collab) [inline Philippines/bago/aral/aral.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: Philippines/bago/bagongpanimula.md (hunk 4 above)
Comment (casey-collab) [inline Philippines/bago/bagongpanimula.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**